firehose/info/info_filler.go

```diff
@@ -4,46 +4,49 @@ import (
 	"fmt"
 	"strings"
 
+	registry "github.com/pinax-network/graph-networks-libs/packages/golang/lib"
 	pbbstream "github.com/streamingfast/bstream/pb/sf/bstream/v1"
-	wellknown "github.com/streamingfast/firehose-core/well-known"
 	pbfirehose "github.com/streamingfast/pbgo/sf/firehose/v2"
+	"github.com/streamingfast/substreams/networks"
 )
 
 var DefaultInfoResponseFiller = func(firstStreamableBlock *pbbstream.Block, resp *pbfirehose.InfoResponse, validate bool) error {
```

Comment:
I'm not sure implementation is correct here. Because we do some prefetching/prepopulating in `start.go`,  it feels like some of the code is redundant.

Like, why do re-set the `resp.BlockIdEncoding` if it's already set? I guess same with `resp.ChainName` and `resp.ChainNameAliases`

Let me know about it @maoueh.

Reply:
You're right, a few things:

1) the blockIDEncoding and ChainNameAliases should NOT be replaced if they are already set
2) the ChainName DOES need to be set with the value from the registry (there's a comment in the code regarding canonical name...)
3) Do not use `chain.ShortName` for the name, instead, use `chain.ID` everywhere.
we want `solana-mainnet-beta` for chain solana, instead of `Solana`.